### DISCUSSION TITLE
multiple mentions are not working in same line

### DISCUSSION THREAD

**Discussion by debamishra14** [ueberdosis/tiptap#895] [General Questions]
I am using tiptap for the last couple of weeks where I found that when I am trying to mention multiple users then in local its working fine but in production, it's not working where I can see the below differences.
Can anyone help me with the reason why it's happening? 

1. all mentioned users are coming in `<span>` tag inside `<p>` which is perfectly getting rendered in local,
2. but in prod, it's not coming as it comes in local though I'm using the same setup

In local:
```
<p class=""><span class="mention" data-mention-id="VXNlck5vZGU6YTA5YmZjOTMtNjcyZC00MmEz" contenteditable="false">@Kumar james</span><span class="mention" data-mention-id="VXNlck5vZGU6MzY5NTM1Y2Y" contenteditable="false">@gopal</span>  </p>
```

In production: After one mention, another `<span>` is not coming
```
<p class=""><span class="mention" data-mention-id="VXNlck5vZGU6MTRl" contenteditable="false">@Kumar james</span>@gopal </p>
```

**Comment by andreasvirkus** [comment]
@DebaPrasad14 this bug is tracked here - https://github.com/ueberdosis/tiptap/issues/874

**Reply by debamishra14** [reply]
@andreasvirkus any workaround for that? It's very urgent for me to get it fixed. I am using this by installing using npm for a vue project.

**Reply by andreasvirkus** [reply]
Yeah, downgrade `tiptap-extensions` to version `1.33.0`

**Reply by debamishra14** [reply]
@andreasvirkus I am using "tiptap": "^1.27.0", "tiptap-extensions": "^1.29.0" but in my local env its working fine but in prod env its not working. And I can see there is nothing getting changed in prod too. (versions remain same in prod as local)

**Reply by andreasvirkus** [reply]
`"tiptap-extensions": "^1.29.0"` means "Give me a version higher than `1.29.0`, which means it will match with the latest version, aka `1.33.2`. You should pin the `tiptap-extensions` package version (until there's a proper fix) 👇 
```
"tiptap": "^1.30.0",
"tiptap-extensions": "1.33.0"
```
